sale_order_product_recommendation/tests/test_recommendation.py

```diff
@@ -6,11 +6,13 @@
 from freezegun import freeze_time
 
 from odoo.exceptions import UserError
+from odoo.tests import tagged
 from odoo.tests.common import Form
 
 from .test_recommendation_common import RecommendationCase
 
 
+@tagged("post_install", "-at_install")
```

Comment:
Why do you need this? It's better to not do it if not needed.

Reply:
Hi @pedrobaeza,

The reason I used that decorator was to avoid **NotNullViolation**.
Traceback:
psycopg2.errors.NotNullViolation: null value in column "warehouse_id" of relation "sale_order" violates not-null constraint

Reply:
But that's something on your end when you repeat the tests once the module is installed. Please remove it.